src/firebase/firebaseBookClub.ts

```diff
@@ -163,6 +166,43 @@ async function getBookClubDocument(bookClubId: string) {
   }
 }
 
+async function getBookClubsByModerator(moderatorId: string){
```

Comment:
I noticed that there is no pagination, if you want I can add it to my PR.

Reply:
Why do we need that? It is an extremly rare case that a user has that many clubs, isn't it? Or do we need this because of something else? You can add it in your PR then, if that is okay?

Reply:
It is not a must, for sure. I just remembered that the original method had pagination.
We can keep this method without pagination for the time being.